client.go

```diff
@@ -71,12 +71,17 @@ func (m *Client) listenWrite() {
 func (m *Client) listenRead() {
 	for {
 		msg, err := m.conn.Receive()
+
+		// This block is error handling
 		if err == io.EOF {
 			m.sv.OnDelClient(m)
 			return
 		} else if err != nil {
 			m.sv.OnError(err)
-		} else if msg != nil {
+		}
```

Comment:
@pengyudu Please add `return` statement to above. 
And please remove these comments below. These are just examples. Sorry 🙏 

```go
// This block is ...
```

Reply:
@pengyudu Sorry, it should not add the `return` statement. I missed.